reference/functions/getuserinfo.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: getuserinfo
+published: true
+tags: [reference, user functions, functions, getuserinfo, users, uid, gid, gecos, homedir, shell]
+---
+
+[%CFEngine_function_prototype(optional_uidorname)%]
+
+**Description:** Return information about the current user or any other, looked up by user ID (UID) or user name.
```

Comment:
They are local in the sense that the system knows about them, so the function never looks in LDAP or anywhere else by itself. I rephrased it.